### DISCUSSION TITLE
Can I ignore those warnings?

### DISCUSSION THREAD

**Discussion by Nutellaeis** [lldap/lldap#879] [Q&A] [answered]
So currently when starting lldap I get the following warnings:

`WARNING: A key_seed was given, we will ignore the server_key and generate one from the seed!`
No idea why this is a warning. It seems that is the intended behaviour when I set this value.

`WARNING: Unsecure default admin password is used.`
I changed the admin password right after the first login. There is no password set in the config so thats why it might complain.

I feel like I can just ignore those but usually if something tagged as `WARNING` it means one should act on it so I wanted to make sure it is safe to ignore those.

**Comment by nitnelave** [comment, marked as answer]
The first warning doesn't exist in the latest version of LLDAP: we do something smarter about detecting probably-invalid configurations and pointing the user to actions to remedy that.

The second warning can just be silenced by setting the admin password to "manually-changed" or anything non-default.

**Reply by nitnelave** [reply]
Note: The latest non-stable version, as of this writing.

**Reply by nitnelave** [reply]
and to answer your question, yes, you can ignore those warnings :)

**Reply by Nutellaeis** [reply]
Thank you for the quick reply. I thought something like that was the case. Then I will just wait to see if the first warning changes with the next updates :)
